README.md

```diff
@@ -1,12 +1,114 @@
 # Hint-Classifier
 
+## Overview 
+
+This is a model that is trained to classify problem statements from LeetCode problems based on what solution features each problem statement contains.
+As of 6/11/2020, we are classifying problem statements that have the following features:
+
+* Lists
+* Sets
+* Stacks
+* Queues
+* Priority Queues
+* Maps
+
+Problem statements are representing using [word embedding model](https://towardsdatascience.com/nlp-extract-contextualized-word-embeddings-from-bert-keras-tf-67ef29f60a7b).
+
+### Data Gathering and Preprocessing
+
+Relevant files: `problem.py`, `parser.py`, `problems.json`
+
+Problem statements and solutions for each problem statement are scraped from LeetCode and stored in `statements.txt` and `solutions.txt`.
+As of 6/11/2020 we currently have 762 LeetCode problem statements with corresponding solutions.
```

Comment:
There are 763 problems not 762. Ik it says 762 in embeddings.py, I just never changed it.